python/ml4ir/applications/classification/tests/test_calibration.py

```diff
@@ -6,20 +6,22 @@
 import tensorflow as tf
```

Comment:
two new tests for model saving/loading with TS added to this file and failure messages added to the previous tests